drink_machines.lua

```diff
@@ -397,6 +417,9 @@ minetest.register_node('drinks:liquid_barrel', {
       if outputstack == 'vessels:glass_bottle' then
          drinks.drinks_liquid_sub(4, 'jbo', 128, pos)
       end
+      if outputstack == 'vessels:steel_bottle' then
```

Comment:
so with elseif's you exit the entire list once a hit has been found. Now each of the conditions is always run.

The whole section can be replaced with a simple table lookup:

(untested, concept code :))
```
local tbl = {
   ["vessels:steel_bottle"] = {name = "jsb", size = 256},
   jsl = {name = "......

...

drinks.dinks_liquid_sub(4, tbl[outputstack].name, tbl[outputstack].size, pos)
```